doc/user/content/connect-sources/cdc-postgres-direct.md

```diff
@@ -108,51 +160,100 @@ As a _superuser_ (`cloudsqlsuperuser`):
+
+{{< tabs >}}
+{{< tab "AWS RDS">}}
+
+1. For each table that you want to replicate to Materialize, set the [replica identity](https://www.postgresql.org/docs/current/sql-altertable.html#SQL-ALTERTABLE-REPLICA-IDENTITY) to `FULL`:
 
     ```sql
-    ALTER ROLE "dbuser" WITH REPLICATION;
+    ALTER TABLE table1 REPLICA IDENTITY FULL;
     ```
 
-    And ensure that this user has the right permissions on the tables you want to replicate:
+    This setting makes sure sufficient information is written to the replication stream for  `UPDATE` and `DELETE` operations.
+
+    <!-- THE LINKED PAGE DOESN'T TALK ABOUT INCREASED CPU. WE SHOULD FIND ANOTHER PAGE TO LINK TO. COMMENTING THIS OUT IN THE MEANTIME. As a heads-up, you should expect a performance hit in the database from increased CPU usage. For more information, see the [PostgreSQL documentation](https://www.postgresql.org/docs/current/logical-replication-publication.html). -->
+
+1. Create a [publication](https://www.postgresql.org/docs/current/logical-replication-publication.html) with the tables you want to replicate:
+
+    _For specific tables:_
 
     ```sql
-    GRANT CONNECT ON DATABASE dbname TO dbuser;
-    GRANT USAGE ON SCHEMA schema TO dbuser;
-    GRANT SELECT ON ALL TABLES IN SCHEMA schema TO dbuser;
+    CREATE PUBLICATION mz_source FOR TABLE table1, table2;
     ```
 
-    **Note:** `SELECT` privileges on the tables you want to replicate are needed for the initial table sync.
-
-1. Set the replica identity to `FULL` for the tables you want to replicate:
+    _For all tables in the database:_
 
     ```sql
-    ALTER TABLE repl_table REPLICA IDENTITY FULL;
+    CREATE PUBLICATION mz_source FOR ALL TABLES;
     ```
 
-    This setting determines the amount of information that is written to the WAL in `UPDATE` and `DELETE` operations.
+    The `mz_source` publication will contain the set of change events generated from the specified tables, and will later be used to ingest the replication stream.
 
-    As a heads-up, you should expect a performance hit in the database from increased CPU usage. For more information, see the [PostgreSQL documentation](https://www.postgresql.org/docs/current/logical-replication-publication.html).
+    We strongly recommend that you **limit** publications only to the tables you need.
 
-1. Create a [publication](https://www.postgresql.org/docs/current/logical-replication-publication.html) with the tables you want to replicate:
+1. Create a user for Materialize:
 
-    _For specific tables:_
+    ``` sql
+    CREATE USER materialize PASSWORD '<password>';
+    ```
+
+1. Grant the user permission to manage replication:
+
+    ``` sql
+    GRANT rds_replication TO materialize;
+    ```
+
+1. Grant the user the required permissions on the tables you want to replicate:
```

Comment:
Not a huge deal, but this says to grant permissions for the tables you want to replicate, but then grants privileges on all tables in the schema. That's probably fine but worth changing the language or the instruction (I'd change the language because enumerating the list of tables you want to select from is annoying).

Reply:
FWIW, in our internal use case (populating ODW Materialize with fivetran data), we don't want to have to go back in and grant permissions on each new table that Fivetran creates in the database for us, so the option of granting permissions on everything in the schema is more convenient. It would be nice to have a suggestion that this is possible, at least.

Reply:
Done.